pkgs/applications/version-management/git-and-tools/bitbucket-server-cli/default.nix

```diff
@@ -0,0 +1,19 @@
+{ lib, bundlerEnv, ruby }:
+
+bundlerEnv rec {
+  name = "bitbucket-server-cli-${version}";
+
+  version = (import gemset).atlassian-stash.version;
```

Comment:
If the package name was the same as the gem name, setting `pname = "atlassian-stash";` would have the same result.